packages/edge-login-ui-react/src/locales/default.js

```diff
@@ -852,24 +852,24 @@ const strings = {
 
   app_support_requested: 'Support Requested',
   app_name: 'Edge',
-  app_url_prefix: 'airbitz',
-  app_homepage: 'https://airbitz.co',
-  app_support_email: 'support@airbitz.co',
+  app_url_prefix: 'edge',
```

Comment:
Where is app_url_prefix used? Did you test that this still works?

Reply:
Searching that yields no results on the code, must be from the old codes. Yup everything works fine